### ISSUE TITLE
Quick pick: Focusing the text box again will not announce its contents

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#93926]
https://github.com/microsoft/vscode/issues/93641

Version: 1.44.0-insider (user setup)
Commit: d7d1147d6b764a48b789000c03088ee9afabcd8a
Date: 2020-03-31T08:23:56.446Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

---

Repro:

1. Ctrl+p
2. "term "
3. down
4. tab (nothing announced, see https://github.com/microsoft/vscode/issues/93916)
5. shift+tab (**nothing announced**)

"term " should be announced as that's what's getting focused but nothing is.

**Comment by bpasero** [comment]
@Tyriar you press tab and then focus back into the input box?

**Comment by Tyriar** [comment]
I tested original issue without a11y mode on. The case this actually calls out closes the quick pick when accessibility mode is on, however the same problem happens when focusing between the text box and the first item. I've updated the original issue for the problem under accessibility mode.

**Comment by isidorn** [comment]
This actually works for me both for NVDA and for VoiceOVer.
The only thing I could reproduce is step 4. where the first tab does not announce anything (though this is up to NVDA and not much we can do). Since VoiceOver announces the focus moved.
Step 5. works just fine for me.

@Tyriar are you doing something special which I am missing?

**Comment by bpasero** [comment]
@isidorn I cannot reproduce with VoiceOver, when I tab back into the input box I get the value announced. I also get something announced when tabbing at first. Probably NVDA is different here.

Since I do not see an immediate fix for March, moving to April to decide how to proceed.

**Comment by isidorn** [comment]
No action needed since you see the same as me. Only reproduces for @Tyriar 
Let's wait for the community feedback. Thus moving out to Backlog.

**Comment by Tyriar** [comment]
Updated NVDA to 2019.3.1 and it still happens.

**Comment by isidorn** [comment]
Nobody other than @tyriar hit this, closing as will not fix.
